src/components/blog-bio/bio.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Blog Bio component
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import {GatsbyImage} from 'gatsby-plugin-image';
+
+import ShortBio from './short-bio';
+import * as styles from './blog-bio.module.scss';
+import PersonSocialMediaLinks from '../person/social-media-links';
+
+const Bio = ({fullName, writtenDate, avatar, email, github, linkedIn}) => {
+  return (
+    <div className={styles.bioContainer}>
+      <div className={styles.avatarColumn}>
+        <GatsbyImage
+          className={styles.avatarImg}
+          alt={fullName || 'Author Picture'}
+          image={avatar}
+        />
+      </div>
+      <div>
+        <p>
+          <ShortBio fullName={fullName}/>
+        </p>
+        <p className={styles.writtenDate}>{writtenDate}</p>
+        <div className={styles.signpostContainer}>
+          <PersonSocialMediaLinks
+            github={github}
+            linkedIn={linkedIn}
+          />
+          <span className={styles.email}>{email}</span>
+        </div>
+        <p style={{ "margin-bottom": 0 }}>datavaluepeople is a group of data-driven people. Through applied machine learning, building automated systems, advising, and education, we create value for businesses, organizations, and humans. Contact us to find out more about how we can help your organisation or if you are interested in the team.</p>
```

Comment:
The `Contact us to find out more about how we can help your organisation or if you are interested in the team.` could maybe be better as: `Get in touch us to speak to us about how we could work with your organisation, or if you are interested in joining our team.`

What you think?

Reply:
Can we make this sentence bold too? Or that a bit extra? I think it worth it.

Reply:
9f1a533 and e2ff88c

I think the blod is ok. not 100% sure on the visual aspect of it but I think it is good to make that more prominiate. 

One other option is we add a new line to make it more prominate: 
![image](https://user-images.githubusercontent.com/4256007/212260191-8d78c160-00e3-4890-a9a6-343c4cbe3cf0.png)

Reply:
There's a typo/mistake in "Get in touch **us** to speak to us...". Since reading it a few time more, I think it would be better if is was: 
```
Drop us an email to speak to us about how we could...
```
Makes it clearer how to get in touch. 
I think I'd prefer doing it bold and not new line.

Reply:
This is what is on https://dvp-landing-static.web.app/blog/test/hello-world/ - it doesn't have the bold it doesn't look like. Maybe it needs another build & push? 
![image](https://user-images.githubusercontent.com/8318425/213180598-35522f9c-0b02-4ec1-bb2b-39a6d43a91e4.png)

Reply:
Fixed now and has been deploy to staging2 (dvp-landing-static)

https://github.com/datavaluepeople/dvp-landing-static/pull/66/commits/79c1e7b7dca00ae5cba633de2c01ad6311db16c8